pom.xml

```diff
@@ -8,7 +8,7 @@
     <artifactId>opentracing-grizzly-ahc</artifactId>
     <version>0.0.1-SNAPSHOT</version>
 
-    <name>${project.groupId}:${project.artifactId}</name>
+    <name>SpecialAgent Rule for Grizzly AsyncHttpClient</name>
```

Comment:
Can we keep the previous name?

Reply:
This should be a descriptive name, because it is used in SEO.

Reply:
https://github.com/opentracing-contrib/java-jdbc/blob/master/pom.xml#L23
https://github.com/opentracing-contrib/java-grpc/blob/master/pom.xml#L23

I tried to follow what other projects had. Though I'm not opposed to name it differently I don't think the project is SpecialAgent specific, so I'm thinking it should not be part of the name.

Reply:
Yes, I agree. I believe the most proper name would be: `<name>OpenTracing Instrumentation Plugin for Grizzly AsyncHttpClient</name>`